code/modules/clothing/spacesuits/rig/suits/station.dm

```diff
@@ -519,3 +519,46 @@ Technomancer RIG
 		/obj/item/rig_module/ai_container,
 		/obj/item/rig_module/storage/large
 		)
+
+/***************************************
+	MU66 Suit
+****************************************/
+/obj/item/rig/sea
```

Comment:
If it's meant to be a moderate suit vs a combat suit, make sure to actually define some values for ablative.

Reply:
What values do you think should be for it?